Utils/slack_alerts/base.py

```diff
@@ -0,0 +1,145 @@
+from typing import List, Dict
+
+from slack_sdk.web import WebClient, SlackResponse
+from slack_sdk.errors import SlackApiError
+from .config import SlackConfig
+
+from slack_utilities import logging
+
+logger = logging.setup_logging()
+
+
+class SlackMessaging(SlackConfig):
+    def __init__(self, token):
+        self.client = WebClient(token=SlackConfig.SLACK_TOKEN)
```

Comment:
It would be good too to send an email alert here if the SLACK_TOKEN is expired or otherwise unusable.